### PR TITLE
[jsk_fetch_startup/kitchen] Use smach to send mail with smach_to_mail.py

### PR DESCRIPTION
**Pull request by tkmtnt7000**
This PR splits feature in #303.
This PR enables us to take pictures, write description and send mail (half-automatically) with smach_to_mail.py.
It refrects https://github.com/jsk-ros-pkg/jsk_robot/pull/1558#issuecomment-1209053331

### CODE REVIEW

**Comment by tkmtnt7000** [comment]
https://user-images.githubusercontent.com/67531577/185872648-ae023dbb-e0a2-4de4-ba47-6dc26a0e9589.mp4

**Comment by tkmtnt7000** [comment]
Ready to review.

**Review by knorth55 — APPROVED** [review]

File: jsk_fetch_robot/jsk_fetch_startup/launch/visualize_trash.launch [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<launch>
```
Comment (knorth55) [inline jsk_fetch_robot/jsk_fetch_startup/launch/visualize_trash.launch]: do we need this launch?

Comment (tkmtnt7000) [reply jsk_fetch_robot/jsk_fetch_startup/launch/visualize_trash.launch]: Sorry, I mistakenly added it.
I removed it in 5235bdd.

Comment (tkmtnt7000) [reply jsk_fetch_robot/jsk_fetch_startup/launch/visualize_trash.launch]: If I should remove this file from the commit log, I will remove it with force-pushing.

Comment (knorth55) [reply jsk_fetch_robot/jsk_fetch_startup/launch/visualize_trash.launch]: you can do force pushing, if you want.

Comment (tkmtnt7000) [reply jsk_fetch_robot/jsk_fetch_startup/launch/visualize_trash.launch]: Thank you. I removed this from commit log.

**Pull request merged**